modules/scala/scala-interpreter/src/main/scala/almond/ScalaInterpreter.scala

```diff
@@ -494,10 +494,20 @@ final class ScalaInterpreter(
     )
   }
 
-  override def inspect(code: String, pos: Int, detailLevel: Int): Option[Inspection] = {
-    // TODO Use ammonite.repl.tools.source.load to return some details here
-    None
-  }
+  override def inspect(code: String, pos: Int, detailLevel: Int): Option[Inspection] =
+    ammInterp.compilerManager.info(
+      pos,
+      frames().head.imports.toString(),
+      code
+    ) match {
+      case Success(s) =>
+        Some(Inspection.fromDisplayData(
+          DisplayData.text(s)
```

Comment:
You might be able to get a slightly nicer `toString` with this here https://github.com/scalameta/metals/blob/cec8b98cba23110d5b2919d9879c78d3b0146ab2/metaserver/src/main/scala/scala/meta/languageserver/providers/HoverProvider.scala#L34